requirements.txt

```diff
@@ -1,5 +1,6 @@
 cython>=0.29.24
 Flask>=3.0.1
+Flask-Limiter[redis]
```

Comment:
This PR is perfect as is. Can you just add a version requirement on the new dependency? `Flask-Limiter[redis]>=3.8.0`